services/core-api/tests/factories.py

```diff
@@ -938,7 +938,7 @@ class Meta:
 
     class Params:
         minimal = factory.Trait(
-            mine_no=None,
+            mine_no='123456',
```

Comment:
also never like how test emails go out like "Mine No: <empty>" because I'm not sure if it's working properly, so put in a default value here. And made the AmsFinalApplication factory use the correct authorization type below. 😅